### ISSUE TITLE
Admins do not 404 for operations they are not a member of

### ISSUE THREAD

**Issue by jrozner** [ashirt-ops/ashirt-server#16]
When a super admin force browses to an operation that they are not a member of they are able to load the page with a timeline though the media and left rail fails to load. From here they are able to interact with the operation and read content or force browse to the operation settings view where they can perform any action including adding themselves as a member with whatever permission is desired.

**Comment by JoelAtDeluxe** [comment]
This is a side effect due to the way we implemented super admins being able to access an operation's settings page via the admin page. Essentially, we added in a check for permissions on a number of services. Roughly:

```python
if super_admin:
  allow
else:
  do_actual_permission_check()
```

I see 3 possible fixes right now (in reality, there could be more or fewer):

1. Remove the bypass above specifically for list evidence for operation. I'm not sure why this is there currently, and it may be required, but if not, we can achieve a kind of 404-lite experience (still able to view the page, but every sub-component renders a 404 error)
2. Add some kind of tag or indication on every request done via an admin that the admin bypass should be checked, which hopefully will prevent this situation, except when the admin specifically tries to do this.
3. Duplicate all of the endpoints to have admin versions and non-admin versions. 

There are potentially issues with each of these, so we may need to reconsider the whole feature. 

Two notes though: Currently, while super admins can view evidence (to a degree -- just tags and descriptions as far as I can tell), they cannot edit that evidence. Also admins should be able to view the settings page of any operation, per a previous request. How they get their is kind of up to them -- it's the same page regardless. Do we need two proper versions to prevent the above situation, knowing that super admins  have this ability anyway?

**Comment by JoelAtDeluxe** [comment]
Regarding list evidence for operation:  This exists because this is the way we calculate how many pieces of evidence are tied to a particular tag: We get a list of all evidence, then do some typescript work to reduce that evidence into a count for each piece of evidence. I think we can revise this specific functionality to instead hit a different endpoint and have the backend do this work for us (either in Go, or hopefully in sql), and then removing the bypass on this service would be possible. This should actually make navigating to an existing, but non-accessible operation, look the same as a non-existing operation. 

However, as noted above, this would not affect settings, as the settings menu is specifically allowed to be accessed by super admins.

**Comment by JoelAtDeluxe** [comment]
This is addressed in PR #28 

Note that this only affects manually browsing to the operation / evidence listing page. Super admins, by design, are still allowed to manually navigate to an operation settings page.
